processor/src/main/java/org/mapstruct/ap/internal/model/VirtualField.java

```diff
@@ -0,0 +1,42 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
```

Comment:
We should update the header template, more than 50% of 2018 are over :)

Reply:
yep.. there should be an issue for it.. time's progressing fast... too fast..